.env

```diff
@@ -1 +1,2 @@
-DUNE_API_KEY=tuBkOcda6ymMvmnpIX9Qer9Xb5hJ2rf8
\ No newline at end of file
+DUNE_API_KEY=tuBkOcda6ymMvmnpIX9Qer9Xb5hJ2rf8
+NEXT_PUBLIC_GOOGLE_ANALYTICS=
```

Comment:
I checked and this env var is already configured on vercel.

The question then is, why add it now to the env empty?

Reply:
Its empty here just to show users that it is there and how to define it